plugins/gatsby-plugin-whats-new-rss/gatsby-node.js

```diff
@@ -0,0 +1,127 @@
+const fs = require('fs');
+const path = require('path');
+const RSS = require('rss');
+const format = require('date-fns/format');
+const parseISO = require('date-fns/parseISO');
+const unified = require('unified');
+const parse = require('rehype-parse');
+const addAbsoluteImagePath = require('../../rehype-plugins/utils/addAbsoluteImagePath');
+const rehypeStringify = require('rehype-stringify');
+const removeImports = require('remark-mdx-remove-imports');
+const removeExports = require('remark-mdx-remove-exports');
+
+// NOTE: remove-imports and remove-exports are now depreciated
+
+const whatsNewQuery = async (graphql) => {
+  const query = `
+  {
+    site {
+      siteMetadata {
+        title
+        siteUrl
+      }
+    }
+    allMarkdownRemark(filter: {fields: {slug: {regex: "/whats-new/"}}}) {
+      nodes {
+        frontmatter {
+          title
+          releaseDate
+          getStartedLink
+          learnMoreLink
+          summary
+        }
+        fields {
+          slug
+        }
+        htmlAst
+      }
+    }
+  }
+  `;
+
+  const { data } = await graphql(query);
+
+  return data;
+};
+
+const htmlParser = unified()
+  .use(parse)
+  .use(addAbsoluteImagePath)
+  .use(rehypeStringify);
+
+const getFeedItem = (node, siteMetadata) => {
+  const {
+    frontmatter,
+    fields: { slug },
+    htmlAst,
+  } = node;
+  const { title, releaseDate } = frontmatter;
+
+  const html = htmlParser.runSync(htmlAst);
+
+  // time is necessary for RSS validity
+  const date = parseISO(releaseDate);
+  const pubDate = `${format(date, 'EE, dd LLL yyyy')} 00:00:00 +0000`;
+  const link = new URL(slug, siteMetadata.siteUrl).href;
+  const id = Buffer.from(title).toString('base64');
+
+  return {
+    guid: id,
+    title,
+    custom_elements: [
+      { link },
+      { pubDate },
+      { 'content:encoded': html },
+      { description: `ReleasedOn: ${pubDate}.` },
+    ],
+  };
+};
+
+const generateFeed = (publicDir, siteMetadata, reporter, whatsNewNodes) => {
+  const title = `What's new in New Relic`;
```

Comment:
Nitpick, and correct me if I'm wrong @austin-schaefer, I think we're not using a `'` with these: `Whats new`.
```suggestion
  const title = `Whats new in New Relic`;
```

Reply:
What's new should have the apostrophe. Is there a reason that doesn't work in this context?